code/renderers/react/package.json

```diff
@@ -74,6 +74,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.11.5",
+    "@jest/globals": "^29.1.2",
```

Comment:
This should be `"@jest/globals": "^26.6.2",` to match the version in the rest of the repo, I believe.